lab_2/main.py

```diff
@@ -34,7 +24,32 @@ def fill_lcs_matrix(first_sentence_tokens: tuple, second_sentence_tokens: tuple)
     :param second_sentence_tokens: a tuple of tokens
     :return: a lcs matrix
     """
-    pass
+    lcs_matrix = []
+
+    if not isinstance(first_sentence_tokens, tuple) or not isinstance(second_sentence_tokens, tuple) or (first_sentence_tokens == ()) or (second_sentence_tokens == ()):
+        return []
+
+    for token in first_sentence_tokens:
+        if isinstance(token, str):
+            lcs_matrix = create_zero_matrix(len(first_sentence_tokens), len(second_sentence_tokens))
```

Comment:
Do you need to create a matrix on each iteration?
You should call the function `create_zero_matrix` only once